netbout-web/src/test/java/com/netbout/rest/bout/TkInviteTest.java

```diff
@@ -88,4 +91,46 @@ public void invitesUserByEmail() throws Exception {
             throw ex;
         }
     }
+
+    /**
+     * TkInvite should throw an RsFailure when invite alias is too long.
+     * @throws Exception If there is some problem inside
+     */
+    @Test (expected = RsFailure.class)
+    public void inviteWithTooLongAlias() throws Exception {
```

Comment:
@jhyle Test name does not describe the scenario at all. Let's call it `failsWhenInvitingWithTooLongAlias`